docs/sources/auth/saml.md

```diff
@@ -91,14 +100,18 @@ metadata_valid_duration = 48h
 
 And here is a comprehensive list of the options:
 
-| Setting                   | Required | Description                                                                    | Default |
-|---------------------------|----------|--------------------------------------------------------------------------------|---------|
-| `eanbled`                 | No       | Whenever SAML authentication is allowed                                        | `false` |
-| `certificate|_path`       | Yes      | Base64-encoded string or Path for the SP X.509 certificate                     |         |
-| `private_key|_path`       | Yes      | Base64-encoded string or Path for the SP private key                           |         |
-| `idp_metadata|_path`      | Yes      | Base64-encoded string or Path for the IdP SAML metadata XML                    |         |
-| `max_issue_delay`         | No       | Duration, since the IdP issued a response and the SP is allowed to process it  | `90s`   |
-| `metadata_valid_duration` | No       | Duration, for how long the SP's metadata should be valid                       | `48h`   |
+| Setting                     | Required | Description                                                                                        | Default                                                     |
+|-----------------------------|----------|----------------------------------------------------------------------------------------------------|-------------------------------------------------------------|
+| `eanbled`                   | No       | Whenever SAML authentication is allowed                                                            | `false`                                                     |
+| `certificate                | _path`   | Yes                                                                                                | Base64-encoded string or Path for the SP X.509 certificate  |
```

Comment:
We have a rogue `|` here that makes _path end up in the Required column and everything else pushed a column as well.

Perhaps try without the `|` (which could be considered a bit technical) and go with something like this: `certificate` or `certificate_path`

Reply:
Same goes for the other similar cases.